.github/config.yaml

```diff
@@ -0,0 +1,4 @@
+todo:
+  # Our CRDs include K8S core objects which has TO-DOs in their comments
```

Comment:
[minor] Not only this, but let's assume we include some TODOs to the CRDs. Then the bot will pick them up twice: once in the ..._types.go, and once here in the generated files.

Reply:
i'm not sure what is your suggestion here.

in the `api/` dir there can be legit TODO, which is related to our project.

Reply:
The suggestion is to replace the comment to something like: "the CRDs are generated files whose TODO comments should be excluded"